### PR TITLE
Update deployment guide to remove git-flow

### PR DESCRIPTION
**Pull request by benhutton**

### CODE REVIEW

File: workflow/deploying/README.md [hunk 1]
```diff
@@ -115,6 +68,10 @@ How to deploy to desiringgod.org's production server.
   $ heroku restart
   ```
 
-1. Close the release PR and delete the release github branch.
-
 1. Announce the deploy to the team, sharing a link to the release notes.
+
+## Known Liabilities/Difficulties
+
+1. What happens when someone merged a PR during pre-deploy QA? Do we start over? Do we exclude it? How then do we apply fixes? Do we only include it if we need to fix anything?
```
Comment (gaganawhad) [inline workflow/deploying/README.md]: @benhutton These are good questions. I wonder if it helps to note this in a deploy guide though. I guess we can convert this into a workable action point - ask the team to stop merges to master, until we resolve this issue, and make this into an idea, or a card, or note it somewhere else.

File: workflow/deploying/README.md [hunk 2]
```diff
@@ -7,88 +7,41 @@ How to deploy to desiringgod.org's production server.
...
-  ```
-
-1. Create a pull request on github **against master** using a link like this:
-
-  ```
-  https://github.com/desiringgod/desiring-god-new/compare/master...release/<version-number>
   ```
 
-1. Deploy that release branch to staging (you can use the Manual Deploy Web Interface on the heroku app).
+1. Deploy master branch to staging (you can use the Manual Deploy Web Interface on the heroku app).
```
Comment (gaganawhad) [inline workflow/deploying/README.md]: @benhutton I would like this step to mirror this one: https://github.com/desiringgod/guides/pull/1/commits/829ebbb864a8d17f64ecbba7e723a8283c215d1a#diff-55824976e0c3f458c5f1da8e5c0104efR48. Any reason not to use `bin/rake deploy:staging` here?

Comment (gaganawhad) [inline workflow/deploying/README.md]: @benhutton Should 'run any post deploy tasks' be noted here? Same question for `pre-deploy` tasks.

**Comment by benhutton** [comment]
@gaganawhad take another look.

I've moved a warning inline with the rest of the steps. The other ideas in that "liabilities" section I believe are now well known enough and under discussion and dealt with by a couple of prodpad ideas.

**Comment by gaganawhad** [comment]
@benhutton Updates to deploy guide look great! Mind squashing them? I'll merge them in soon after.

**Comment by benhutton** [comment]
@gaganawhad done!

**Pull request merged**